aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -106,7 +109,7 @@ public static String testMultiRegionAccessPointAlias() {
     return System.getenv("AWS_TEST_MULTI_REGION_ACCESS_POINT_ALIAS");
   }
 
-  public static void cleanS3Bucket(S3Client s3, String bucketName, String prefix) {
+  public static void cleanS3GeneralBucket(S3Client s3, String bucketName, String prefix) {
```

Comment:
nit: should we keep this method name as is and change the S3 Express bucket name as S3 Express bucket as `Directory Bucket` to be aligned with public AWS documentation https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-express-differences.html

Reply:
What do you call the existing buckets then? Looks like it is called "general purpose bucket"? I am okay renaming this method to `cleanS3GeneralPurposeBucket` but I agree the other method should be `cleanS3DirectoryBucket`